docs/coding-guidelines.md

```diff
@@ -78,3 +78,45 @@ Example:
  */
 import VisualEditor from '../visual-editor';
 ```
+
+## PHP
+
+We use
+[`phpcs` (PHP\_CodeSniffer)](https://github.com/squizlabs/PHP_CodeSniffer)
+with the
+[WordPress Coding Standards ruleset](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards)
+to run a lot of automated checks against all PHP code in this project.  This
+ensures that we are consistent with WordPress PHP coding standards.
+
+When making any changes to the PHP code in this project, it's recommended to
+install and run `phpcs` on your computer.  This is a step in our Travis CI
+build as well, but it is better to catch errors locally.
+
+You will need to install `phpcs` version 2.9.x, because the 3.x versions are
+not yet compatible with the WordPress coding standards.  For more information see
+[this issue](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/718).
+
+The easiest way to get `phpcs` is to download the .phar archive from the latest
```

Comment:
I wonder if it would be easier to make `phpcs` 2.9.x a Composer dependency instead? If we add a `composer.json` which has `wp-coding-standards/wpcs` among its dev-requirements then all a contributor (and Travis CI alike) would need to do is `composer install` and then do `./vendor/bin/phpcs`.